app/uk/gov/hmrc/economiccrimelevyreturns/navigation/UkRevenuePageNavigator.scala

```diff
@@ -59,10 +59,20 @@ class UkRevenuePageNavigator @Inject() (
       case _       => Future.successful(routes.NotableErrorController.answersAreInvalid())
     }
 
-  private def clearAmlActivityAnswers(eclReturn: EclReturn, mode: Mode)(implicit request: RequestHeader): Future[Call] =
+  private def clearAmlActivityAnswersAndRecalculate(eclReturn: EclReturn, mode: Mode)(implicit
+    request: RequestHeader
+  ): Future[Call] =
+    // TODO: I cannot quite decide whether we actually need to clear and recalculate if in NormalMode or just navigate to the amount due page?
+    //       As we *should* never already have AML activity answers when in NormalMode (unless we dopped out earlier having provided answers and are now resuming within the TTL period?)
+    //       Unconditionally clearing and recalculating might be safer but it just feels like overkill.
```

Comment:
I think it's fine to do what you are doing here to keep things consistent - and this comment can be removed?